src/pages/docs/installation/octopus-server-linux-container/octopus-in-kubernetes.mdx

```diff
@@ -218,7 +226,7 @@ Most of the YAML in this guide can be used with any Kubernetes provider. However
 To find out more about storage classes, refer to the [Kubernetes Storage Classes](https://kubernetes.io/docs/concepts/storage/storage-classes/) documentation.
 :::
 
-Whilst it is possible to mount external storage by manually defining Persistent Volume definitions in YAML, Cloud providers offering Kubernetes managed services typically include the option to dynamically provision file storage based on persistent volume claim definitions. 
+While it is possible to mount external storage by manually defining Persistent Volume definitions in YAML, Cloud providers offering Kubernetes managed services typically include the option to dynamically provision file storage based on persistent volume claim definitions. 
```

Comment:
Nit: Consider changing to be less formal, since you're also changing from whilst.
```suggestion
While it's possible to mount external storage by manually defining Persistent Volume definitions in YAML, Cloud providers offering Kubernetes managed services typically include the option to dynamically provision file storage based on persistent volume claim definitions. 
```